docs/how-to/peering-with-content-providers.md

```diff
@@ -69,7 +69,11 @@ To have your platform added to this list, please [open a PR to edit this page](h
 
 ### NFT.Storage
 
-See [nft.storage/PEERS](https://github.com/nftstorage/nft.storage/blob/main/PEERS).
+| Peer ID | Addresses |
+| ------- | --------- |
+|`12D3KooWCRscMgHgEo3ojm8ovzheydpvTEqsDtq7Vby38cMHrYjt`|`/dns4/nft-storage-am6.nft.dwebops.net/tcp/18402`|
```

Comment:
I tried pinging these unsuccessfully.

Reply:
Related: https://github.com/ipfs/ipfs-docs/issues/1429

*all* of the addresses don't seem resolvable